plugins/modules/cloudfront_distribution.py

```diff
@@ -1444,7 +1444,7 @@ def delete_distribution(client, module, distribution):
 
 def update_distribution(client, module, config, distribution_id, e_tag):
     try:
-        return client.update_distribution(DistributionConfig=config, Id=distribution_id, IfMatch=e_tag)['Distribution']
+        return client.update_distribution(aws_retry=True, DistributionConfig=config, Id=distribution_id, IfMatch=e_tag)['Distribution']
```

Comment:
This is just one of the calls.  What I tend to find is that if one call is going to hit the issue, most of them will.  I'd recommend adding this to all of the `client.<something>` calls I think they're all between lines 1420 and 1471

Reply:
I don't have a way to test them all, but I will yes